frontend/src/container/Home.jsx

```diff
@@ -133,17 +134,17 @@ export class Home extends React.Component {
   }
 
   handleClickCheckbox = () => {
-    this.setState(prevState => ({ hidePopupCheckbox: !prevState.hidePopupCheckbox }))
+    this.setState({ hidePopupCheckbox: !this.state.hidePopupCheckbox })
```

Comment:
why this change ? This is React bad practice

Reply:
Resolving this, @gbonaspetti already raised this issue.